core/swift3Action/swift3runner.py

```diff
@@ -47,10 +48,18 @@ def epilogue(self, init_message):
             main_function = init_message['main']
         else:
             main_function = 'main'
+        #make sure there is a main.swift file
+        open(DEST_SCRIPT_FILE, 'a').close()
 
         with codecs.open(DEST_SCRIPT_FILE, 'a', 'utf-8') as fp:
+            os.chdir(DEST_SCRIPT_DIR)
+            for file in glob.glob("*.swift"):
+                if file != "Package.swift" and file != "main.swift" and file != "_WhiskJSONUtils.swift" and file != "_Whisk.swift":
```

Comment:
Not really worth changing now, but if this list gets any longer, you might do something like:

```python
if file not in ["Package.swift", "main.swift", "_WhiskJSONUtils.swift, "_Whisk.swift"] :
```